homeassistant/components/mqtt/client.py

```diff
@@ -655,7 +709,7 @@ async def _async_resubscribe(self) -> None:
         """Resubscribe on reconnect."""
         # Group subscriptions to only re-subscribe once for each topic.
         keyfunc = attrgetter("topic")
-        await self._async_perform_subscriptions(
+        await self._async_queue_subscriptions(
```

Comment:
Can this be refactored such that the debouncer is not at all involved when reconnecting?